DOCS-2134: Fix order of TypeScript steps in rover square tutorial (#2845)
viamrobotics/docs: docs/tutorials/get-started/try-viam-sdk.md

Open a web browser and visit `localhost:8000`.
You should see a disabled button that says `Click me`.

> Open the developer console to see the console output.

Author: ```suggestion
Open the developer console to see the console output including the names of your rover's resources.
```

> Open the developer console by right-clicking the page and selecting `Inspect`, then selecting the console tab.

Reviewer: it's different between different browsers, so I think this may be more generally applicable
```suggestion
Open the developer console to see the console output.
```
